.travis.yml

```diff
@@ -73,6 +73,6 @@ jobs:
         # introduce some new warnings and when testing one such warning appeared in the ethcontract
         # generated contract code which caused the whole file to be printed as part of the warning
         # message which made the build fail (probably travis related).
-        - env CARGO_INCREMENTAL=0 RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Awarnings" cargo test
-        - ./grcov --branch --ignore-not-existing --llvm --ignore "/*" target/debug/ --output-path coveralls.json --output-type "coveralls+" --source-dir . --service-name travis-pro --service-job-id $TRAVIS_JOB_ID
+        - env CARGO_INCREMENTAL=0 RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Awarnings" cargo +nightly test
```

Comment:
The nightly in `cargo +nightly test` shouldn't be necessary but also doesn't hurt. Just mentioning.

Reply:
I added it so that I can easily copy paste the command locally where I have multiple envs installed.